omod/src/main/webapp/fragments/header.gsp

```diff
@@ -72,75 +72,102 @@
             });
 
             <% if (enableUserAccountExt) { %>
-            jq('.identifier').hover(
-                function(){
-                    jq('.appui-toggle').show();
-                    jq('.appui-icon-caret-down').hide();
-                },
-                 function(){
-                    jq('.appui-toggle').hide();
-                    jq('.appui-icon-caret-down').show();
-                 }
-            );
+            var event = ('ontouchstart' in window) ? 'click' : 'mouseenter mouseleave';
+
+            jq('.identifier').on(event,function(){
+                    jq('.appui-toggle').toggle();
+                    // jq(this).toggleClass("appui-toggle");
```

Comment:
What's the use of these commented code?

Reply:
It was a toggle I wrote and afterwards gave a common toggle for both drop down toggle and icon caret down toggle.